lib/jellyfish/room.ex

```diff
@@ -376,6 +382,9 @@ defmodule Jellyfish.Room do
     :ok
   end
 
+  def registry_id(room_id),
+    do: {:via, Registry, {Jellyfish.RoomRegistry, room_id}}
```

Comment:
During testing another approach with storing webhook_url in Registry, I found out that we create [`registry_id` from hand in `RoomService`](https://github.com/jellyfish-dev/jellyfish/blob/a4dd56e4efa4a241192f178c90c3bc67e95d0f2b/lib/jellyfish/room_service.ex#L225), which is error-prone because if we change how registry_id is made in `Room` we could forget to change it in `RoomService`

Reply:
Ah, now I see, it became public. Let's move this then to the top, before `start` function as now it's part of the public API